helm/values.yaml

```diff
@@ -0,0 +1,92 @@
+image:
+  repository: # Docker repository to pull the image from
+
+ingressRoute:
+  enabled: true # Boolean indicating whether to generate Traefik ingress routes
+
+app:
+  name: data-connect-trino
+  configVersion: config-version
+  secretHash: secrets-hash
+  memoryLimit: "1Gi"
+  memoryRequest: "1Gi"
+  oauth:
+    clientSecretName: data-connect-trino-client-credentials # Secret name that contains the oauth client for all app interactions to external services e.g. data-connect-trino-client-credentials
+  spring:
+    profile: "wallet-auth,metrics-enabled,gcp" # Comma separated list of spring profiles to activate for this application
+    datasource:
+      hikari:
+        maxPoolSize: "5" # Maximum number of connections in the database pool for this application
+
+dataConnectTrino:
+  appName: data-connect-trino # Host portion of the data-connect-trino URL to be used for authorization e.g. data-connect-trino
+
+wallet:
+  appName: wallet # Host portion of the wallet URL to be used for authorization e.g. wallet
+  adminKubeKey: wallet-cloud-init-client-credentials # Name of k8s secret containing wallet clientId and clientSecret
+
+auditLogService:
+  appName: audit-log-service # Host portion of the audit-log-service URL e.g. audit-log-service
+  enabled: false
+
+indexingService:
+  appName: indexing-service # Host portion of the indexing-service URL to be used for authorization e.g. indexing-service
+  resourceUrl: # resourceUrl comes from global values and should not be set except in very specific cases
```

Comment:
Is there reason to have these empty values here? Because for other services they are not specified.